pages/[contract]/[tokenId]/index.tsx

```diff
@@ -153,7 +155,11 @@ const Index: NextPage<Props> = ({ collectionId, tokenDetails }) => {
         </div>
       </div>
       <div className="col-span-full mb-4 space-y-4 px-2 md:col-span-4 md:col-start-5 lg:col-span-5 lg:col-start-7 lg:px-0 2xl:col-span-5 2xl:col-start-7 3xl:col-start-9 4xl:col-start-11">
-        <Owner details={token} bannedOnOpenSea={bannedOnOpenSea} />
+        {moonbird?.nesting ? (
+          <MoonbirdCard token={token.token} moonbird={moonbird} />
```

Comment:
About naming the props "details" instead of "token". I see one consistency in original code - when {token} is propagaded the property is "details", but when {token.token} is propagated its called "token".

Two examples:
<Owner details={token} bannedOnOpenSea={bannedOnOpenSea} />
<TokenAttributes token={token?.token} collection={collection} />